tests/Notification/AbstractNotificationTest.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace OpenEuropa\EPoetry\Tests\Notification;
+
+use OpenEuropa\EPoetry\Tests\AbstractTest;
+
+/**
+ * Class AbstractNotificationRequestTest.
```

Comment:
Let's either remove of add a more explanatory comment here.

Reply:
Done.